spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/PropertiesMeterFilterTests.java

```diff
@@ -115,14 +116,6 @@ public void acceptWhenHasHigherEnableTrueExactEnableFalseShouldReturnDeny() {
 				.isEqualTo(MeterFilterReply.DENY);
 	}
 
-	@Test
-	public void acceptWhenHasAllEnableFalseShouldReturnDeny() {
```

Comment:
This test should be kept and should continue to pass. We only want to drop support for `all` when configuring the SLA. It should be kept for controlling which meters are enabled.